tools/rimage/src/module.c

```diff
@@ -252,10 +252,10 @@ static void sections_info_add(struct module_sections_info *info, struct module_s
  */
 static void sections_info_finalize(struct module_sections_info *info)
 {
-	info->file_size = info->end - info->start;
+	size_t size = (info->end > info->start) ? info->end - info->start : 0;
```

Comment:
this is from #8531?

Reply:
You got me! That's why it's a draft, because I'm waiting for https://github.com/thesofproject/sof/pull/8531 to be merged so it doesn't look like a change 😄